releasenotes/notes/update-cr-hamtomo-with-duration-380da3452045cd0c.yaml

```diff
@@ -0,0 +1,34 @@
+---
+features:
+  - |
+    :class:`.CrossResonanceHamiltonian` experiment and its subclass now accept
+    ``durations`` with default values. Note that values should be provided in
+    units of sec rather than samples, and must include pulse ramps at edges.
+    Default values with linear increment are generated according to new experiment options,
+    ``min_duration``, ``max_duration`, and ``num_durations``, when the durations
+    is not explicitly provided. Default value is chosen by assuming a
+    typical ZX rate around 1 MHz in recent IBM Quantum backends.
```

Comment:
```suggestion
    is not explicitly provided. The default values are chosen by assuming a
    typical ZX rate of around 1 MHz which is typical for IBM Quantum backends.
```

Reply:
done in https://github.com/Qiskit/qiskit-experiments/pull/933/commits/d4c9f4696e789b72072b2514332714977a821769